lib/matchers/matchers.ts

```diff
@@ -80,7 +80,26 @@ const solidity = (chai: any, utils: any) => {
     const derivedPromise = promise.then((tx: any) =>
       contract.provider.getTransactionReceipt(tx.hash)
     ).then((receipt: any) => {
-      const {topic} = contract.interface.events[eventName];
+      const eventDescription = contract.interface.events[eventName];
+
+      if (eventDescription === undefined) {
+        const isNegated = this.__flags.negate === true;
```

Comment:
I'm not sure this is the way to go when extending `chai` with a plugin. Couldn't find much information in the docs.